Updating SOC2 compliance status (#20300)
netdata/netdata: docs/security-and-privacy-design/README.md

---

### SOC 2 Compliance

> Netdata is SOC2 Type 1 compliant as of 14-May-2025 across the following Service Criteria:

Reviewer: ```suggestion
Netdata achieved SOC2 Type 1 compliance on May 14, 2025 for these Service Criteria:
```
